src/command_modules/azure-cli-storage/tests/test_storage_blob_scenarios.py

```diff
@@ -131,7 +126,65 @@ def get_account_key(self, group, name):
                         .format(name, group)).output
 
 
-if __name__ == '__main__':
-    import unittest
+class StorageBlobUploadLiveTests(LiveTest):
+    @live_only()
```

Comment:
Why is this necessary?  Shouldn't the fact that it's a LiveTest enforce the "live only" behavior?